singularity-eos/eos/eos_SpinerEOSDependsRhoSie.hpp

```diff
@@ -0,0 +1,731 @@
+#ifndef _EOS_SPINEREOSDEPENDSRHOSIE_HPP_
+#define _EOS_SPINEREOSDEPENDSRHOSIE_HPP_
+
+#include <singularity-eos/eos/eos_SpinerEOSDependsRhoT.hpp>
```

Comment:
Even if you have sie depend on rho-T, you should include the shared header files that sie depends on. It can be confusing to have to look into a different file to see which standard utilities/headers are included.